File: tests/annotated/Sample.java
// File: Sample.java

//  Recommendation (Severity 7): Method name 'setWay' is unclear and doesn't reflect the functionality.
//  Rename method to 'initializeMap'.
public class Sample {
    
    private void setWay(int[][] map) {
        // Bad method name, unclear logic
        for (int i = 0; i < map.length; i++) {
            for (int j = 0; j < map[0].length; j++) {
                map[i][j] = 1;
            }
        }
    }
//  Recommendation (Severity 2): Unused method 'unusedHelper(int)'.
//  Remove unused code or refactor for reuse.

    private int unusedHelper(int x) {
        return x * 42;
    }

    public static void main(String[] args) {
        System.out.println("Running...");
    }
}

File: tests/annotated/sample.py
#  Recommendation (Severity 8): File name 'tests/sample.py' suggests this is a test file, but it contains non-test code. Consider renaming or moving.
#  
# tests/sample.py

#  Recommendation (Severity 4): Unused variable 'y'.
#  Remove or use the variable.
def DoStuff():
    x = 5
#  Recommendation (Severity 6): Unreachable code. The function always returns without executing this logic.
#  Remove dead code.
    y = 10
    # unnecessary logic
    return
#  Recommendation (Severity 5): Unused function 'unused'.
#  Remove or refactor to use the function.

def unused():
    pass